L.Map.Sync.js

```diff
@@ -34,6 +44,9 @@
                 this._syncMaps.forEach(function (synced, id) {
                     if (map === synced) {
                         self._syncMaps.splice(id, 1);
+                        if ('undefined' != typeof map.cursor) {
```

Comment:
`map.cursor` will be `undefined`, so you can just juse `if (!map.cursor)`, like [this](https://github.com/turban/Leaflet.Sync/blob/master/L.Map.Sync.js#L21) and [here](https://github.com/turban/Leaflet.Sync/blob/master/L.Map.Sync.js#L31),